src/Input/Input.tsx

```diff
@@ -22,16 +22,16 @@ export interface InputProps {
  * in this component and should be handled in the consuming app.
  *
  */
-const Input: FC<InputProps & ComponentPropsWithoutRef<'input'>> = ({ label, error = '', ...props }) => {
-    return (
+const Input = forwardRef<HTMLInputElement, InputProps & ComponentPropsWithoutRef<'input'>>(
+    ({ label, error = '', ...props }, ref) => (
         <ThemeProvider theme={theme}>
             <StyledLabel error={error}>
                 {label && <span>{label}</span>}
                 <StyledInput error={error} {...props} />
```

Comment:
I think you need to still add the ref as a prop here
` <StyledInput ref={ref} error={error} {...props} />`

Reply:
Done. Thanks, forgot to actually add ref.